build_runner/lib/src/generate/watch_impl.dart

```diff
@@ -76,8 +78,11 @@ class WatchImpl implements BuildState {
     var fatalBuild = new Completer();
     var firstBuild = new Completer<BuildResult>();
     currentBuild = firstBuild.future;
-    var changes =
-        startFileWatchers(packageGraph, _logger, _directoryWatcherFactory);
+    var changes = new PackageGraphWatcher(packageGraph,
```

Comment:
this name reads strangely to me. We aren't watching package graphs, we are watching package and we happen to use a package graph to get set up. Would `PackageWatcher` work? Maybe `PackageDirectoryWatcher`?

Reply:
Since it's an internal class, doesn't bother me too much.

Happy to change it to anything else though.